oh_staff_ui/management/commands/import_projectitems.py

```diff
@@ -0,0 +1,101 @@
+from django.core.management.base import BaseCommand
+from oh_staff_ui.models import ProjectItem, ItemStatus, ItemType
+from django.contrib.auth.models import User
+from csv import DictReader
+import os
+import datetime
+
+
+class Command(BaseCommand):
+    help = "Deletes all existing ProjectItem data, then imports from a CSV file"
+
+    def add_arguments(self, parser):
+        parser.add_argument("filename", type=str)
+
+    def handle(self, *args, **options):
+        # to avoid duplication, start by deleting all existing ProjectItems
+        print("Deleting all existing ProjectItems.")
+        ProjectItem.objects.all().delete()
+
+        # open and read input CSV - assuming it's in the fixtures folder
+        filepath = os.path.join("oh_staff_ui/fixtures", options["filename"])
+        with open(filepath, mode="r") as f:
+            dict_reader = DictReader(f)
+            projectitem_dicts = list(dict_reader)
+        total_rows = len(projectitem_dicts)
+        print(f"Found {total_rows} ProjectItems to import.")
+
+        # start with known top-level ProjectItem
+        # handled outside of main loop due to lack of parent
+        added_items = []
+        root_item = next(
+            item for item in projectitem_dicts if item["ARK"] == "21198/zz0008zh0f"
+        )
+        # create and save ProjectItem for root object
+        p = ProjectItem(
+            ark=root_item["ARK"],
+            create_date=self.format_date(root_item["CREATE_DATE"]),
+            created_by=self.get_or_create_user(root_item["CREATED_BY"]),
+            last_modified_date=self.format_date(root_item["LAST_MODIFIED_DATE"]),
+            last_modified_by=self.get_or_create_user(root_item["LAST_MODIFIED_BY"]),
+            sequence=self.format_sequence(root_item["SEQUENCE"]),
+            status=ItemStatus.objects.get(status=root_item["STATUS"]),
+            title=root_item["TITLE"],
+            type=ItemType.objects.get(type=root_item["TYPE"]),
+        )
+        p.save()
```

Comment:
@ztucker4 I don't understand why this root item gets saved to the database.  I thought we did not want to migrate this single top-level collection record?  But I see it in the database after this runs:
```
>>> p = ProjectItem.objects.get(ark="21198/zz0008zh0f")
>>> p
<ProjectItem: Oral History Collection>
>>> vars(p)
{'_state': <django.db.models.base.ModelState object at 0x7f4f34d64290>, 'id': 20454, 'ark': '21198/zz0008zh0f', 'create_date': datetime.date(2008, 1, 31), 'created_by_id': 2, 'last_modified_date': datetime.date(2009, 6, 1), 'last_modified_by_id': 3, 'parent_id': None, 'sequence': 1, 'status_id': 2, 'title': 'Oral History Collection', 'type_id': 1}
>>>
```